themes/theme-ios11/components/TabBar/components/FavoritesAction/components/FavoritesIconBadge/index.jsx

```diff
@@ -41,9 +48,15 @@ export class FavoritesIconBadge extends Component {
    * @returns {JSX}
    */
   render() {
+    const { showCounter } = this.props.widgetSettings;
     if (this.props.favoritesCount === 0) {
       return null;
     }
+
+    if (showCounter === false) {
+      return <div className={`${style} theme__tab-bar__favorites-icon-badge theme__badge`} />;
```

Comment:
instead of this if here, you do do below in L64:

`return (
      <div className={`${style} theme__tab-bar__favorites-icon-badge theme__badge`}>{showCounter !== false ? number : ''}</div>
    );`
(not tested)